class.jetpack.php

```diff
@@ -4607,6 +4619,40 @@ function authenticate_jetpack( $user, $username, $password ) {
 		return new WP_User( $token_details['user_id'] );
 	}
 
+	/**
+	 * Authenticates REST API via user_tokens passed as GET query parameter
+	 */
+	function authenticate_user_tokens( $user_id ) {
+		$get_token = isset( $_GET[ 'user_token' ] ) ? $_GET[ 'user_token' ] : false;
```

Comment:
Does it have to be a `$_GET` param?
Could it instead be an actual authorization header? Should we support both?